core/src/ops/matmul/mir_quant.rs

```diff
@@ -14,9 +16,9 @@ pub struct QParams {
     pub c_scale: AttrOrInput,
 }
 
-impl QParams {
-    pub fn noop_static(dt: DatumType) -> QParams {
-        QParams {
+impl MatMulQParams {
```

Comment:
I renamed this so that the name "QParams" be left for the parameters of a datum type.